src/components/Editor/Editor.css

```diff
@@ -160,8 +160,12 @@ html[dir="rtl"] .editor-mount {
   color: var(--theme-content-color3);
 }
 
+.debug-expression {
+  background-color: var(--debug-expression-background);
+}
+
 .new-debug-line .CodeMirror-line {
-  background-color: var(--debug-line-background) !important;
+  background-color: transparent !important;
   outline: var(--debug-line-border) solid 1px;
```

Comment:
Maybe move this to the `.debug-expression`? I think it'd look a little nicer rather than the whole line having the outline.